### ISSUE TITLE
Deprecation notice?

### ISSUE THREAD

**Issue by einarq** [module-federation/external-remotes-plugin#6]
Should this plugin be deprecated, in favor of the "promise new Promise" approach thats recommended in the docs?

**Comment by lessfish** [comment]
@einarq Hi, does the plugin still works?

**Comment by SunHuawei** [comment]
This is a much more declarative way to customize the URL in runtime. When using "promise new Promise", I have to copy and modify code into every project which is an "ugly" solution to me.

**Comment by einarq** [comment]
I agree. There probably is a good reason tho I guess. 
Personally I've created some utils for dealing with stuff like this, and sharing it in an npm package on our internal npm feed, so that I don't end up copy-pasting this around.

**Comment by SunHuawei** [comment]
@einarq, yes, that's a good solution. I just prefer not to customize it if possible, in order to reduce the effort of maintenance and to save time for the other team members to understand the solution.
